sys-libs/libcxx/libcxx-11.0.0.ebuild

```
@@ -58,6 +59,11 @@ src_prepare() {
 	eapply "${FILESDIR}/${PN}-3.9-cmake-link-flags.patch"
 
 	llvm.org_src_prepare
+
+	# eprefixify static path references to libc++abi for symbol re-export to
+	# avoid linking against it twice in both /usr/lib and ${EPREFIX}/usr/lib
+	local lcpa=/usr/lib/libc++abi.dylib
+	sed -i -e "s,${lcpa},${EPREFIX}${lcpa},g" lib/CMakeLists.txt
```

```suggestion
	sed -i -e "s^/usr/lib/libc++abi.dylib^${EPREFIX}&^g" lib/CMakeLists.txt || die
```

Please really add `|| die` to all sed calls. Also `,` seems remotely possible in EPREFIX.

---

done